package/com.unity.formats.usd/CHANGELOG.md

```diff
@@ -2,14 +2,24 @@
 
 ## Unreleased
 ### Bug Fixes
-- Fixed an import bug causing instanced primitives not to be sanitized.
-- Disabled plugins on unsupported platforms.
-- Fixed a bug causing PointInstances to be duplicated.
+- Fixed loading of meshes with arbitrary primvars.
+- Fixed regression in animated mesh properties.
+- Fixed timeout when importing complex meshes.
+- Fixed a bug on export where SkelRoot and Skeleton Prim were missing class identifiers.
+- Fixed 'ExportSelectedWithChildren' on Mac.
+- Fixed texture mapping for exported meshes.
+- Fixed  a bug causing PointInstances to be duplicated.
+- Fixed regression in JointsMatch().
+- Fixed scene not being marked dirty after changing load payload flag.
+- Fixed exported Prims not having a type and fixed export for objects with same name.
 - Fixed a bug causing an ApplicationException "all sample times in times and baseTime must either all be numeric or all be default" to be thrown when refreshing a PointInstancer.
-- Optimized triangulation to reduce the chance of a "Timed out while waiting for thread" error on importing a complex mesh.
+- Fixed an import bug causing instanced primitives not to be sanitized.
```

Comment:
We should perhaps briefly define what sanitized means - or is it defined elsewhere?

Reply:
I'll add more detail here, but I think we should define this in the documentation as hidden in the changelog possibly isn't the place for it- I'll add a full explanation of the sanitisation to the new docs I'm preparing.

Reply:
Sounds good.